packages/gallery/src/components/gallery/proGallery/slideshowView.js

```diff
@@ -984,8 +987,24 @@ class SlideshowView extends React.Component {
     ];
   }
 
+
+  getBufferedItems(galleryGroups, container) {
+    const { state, props } = this;
+    const { options } = props;
+    const { activeIndex } = state;
+    const groups = this.props.getVisibleItems(galleryGroups, container);
```

Comment:
[question]
Why didn't you destruct the `getVisibleItems and galleryContainerRef` from `this.props` like you did with `options`